pkg/security/tests/process_windows_test.go

```diff
@@ -23,11 +23,7 @@ func TestBasicTest(t *testing.T) {
 		ID:         "test_basic_rule",
 		Expression: `exec.file.name in ["at.exe","schtasks.exe"]`,
 	}
-	opts := testOpts{
-		enableFIM: true,
-	}
-	test, err := newTestModule(t, nil, []*rules.RuleDefinition{ruleDef}, withStaticOpts(opts))
-	if err != nil {
+	test, err := newTestModule(t, nil, []*rules.RuleDefinition{ruleDef})
 		t.Fatal(err)
```

Comment:
syntaxic error: forgot the `if err != nil {`, resulting in just the `}` being present